packages/playwright-core/src/client/playwright.ts

```diff
@@ -73,4 +82,16 @@ export class Playwright extends ChannelOwner<channels.PlaywrightChannel> {
   static from(channel: channels.PlaywrightChannel): Playwright {
     return (channel as any)._object;
   }
+
+  _browserTypes(): BrowserType[] {
```

Comment:
Do we still need this one?

Reply:
at least for the lines below - I made it private for now.